src/analysis/normed_space/multilinear.lean

```diff
@@ -852,7 +852,7 @@ multilinear_map.mk_continuous
   { to_fun := λ m, linear_map.mk_continuous
       { to_fun := λ x, f x m,
         map_add' := λ x y, by simp only [map_add, continuous_multilinear_map.add_apply],
-        map_smul' := λ c x, by simp only [continuous_multilinear_map.smul_apply, map_smul]}
```

Comment:
all these `simp only` make sure that the simp is not too slow. Could you `squeeze_simp` the new one? (Here and elsewhere, just look for `simp only` on this page).